bounties/level-1/basic-account-viewer.md

```diff
@@ -0,0 +1,44 @@
+# Basic account viewer
+
+## Award
+| Hunter | Reviewer | Type
+| :- | :- | :-
+| 200 XLM | 1 Credits | Continuos
+
+## Context
+https://github.com/tyvdh/stellar-quest-bounties/issues?q=basic-account-viewer.md
+
+## Description
+
+### What is this task?
+
+When building websites ontop of Stellar, there needs to be a way to access a user's Stellar account. This is where web-wallets like [Albedo](https://albedo.link/), [Rabet](https://rabet.io/) and [Freighter](https://www.freighter.app/) come in. They allow websites to easily and securely acces your Stellar account.
+
+Your task is to build a simple website which allows you to sign in using a web-wallet like Albedo, Freighter or Rabet.
+
+The website should show some basic information about the account currently logged in:
+ * The account's assets and their balances should be shown.
+ * The date when the account was created should be shown.
+ * The account who created this account should be shown.
+ 
+### What are the requirements for the bounty hunter?
+* The hunter should be able to build simple websites with Javascript.
+
+### What are the deliverables?
+#### Description
+  - The hunter should provide a link to the repository containing the solution.
```

Comment:
Wouldn't it be nice if the outcoume were some actually running website?

Easiest solution could be as gh-pages (e.g. https://github.com/gitname/react-gh-pages).

Reply:
Yes, that's a good idea.